node/repl.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+// Copyright Joyent and Node contributors. All rights reserved. MIT license.
+
+import { notImplemented } from "./_utils.ts";
+
+class REPLServer {
+  constructor() {
+    notImplemented();
+  }
+}
+export const builtinModules = undefined;
```

Comment:
```suggestion
export const builtinModules = ["assert", "async_hooks", "buffer", "child_process", "cluster", "console", "constants", "crypto", "dgram", "diagnostics_channel", "dns", "domain", "events", "fs", "http", "http2", "https", "inspector", "module", "net", "os", "path", "perf_hooks", "process", "punycode", "querystring", "readline", "repl", "stream", "string_decoder", "sys", "timers", "tls", "trace_events", "tty", "url", "util", "v8", "vm", "wasi", "worker_threads", "zlib"];
```
Which, granted, is a bit of a lie.

Reply:
Looks good to me. Thanks for the suggestion!